docs/Testing.fsx

```diff
@@ -113,7 +120,7 @@ let sampleA = vector [|4.5; 5.1; 4.8; 4.4; 5.0|]
 // calculates a one sample t test with a given sample and the fixed value the sample should be compared with
 let oneSampleTTest = TTest.oneSample sampleA 5.
 
-(*
+(**
```

Comment:
This is an inline comment that should be printed as such. Please remove the second asterisks